### ISSUE TITLE
Does chproxy of clickhouse support vector data ingestion?

### ISSUE THREAD

**Issue by hueiyuan** [vectordotdev/vector#20493]
### A note for the community

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Problem

Hi, all

Now we apply [vector](https://vector.dev/docs/) to ingest data to clickhouse. And we found that corresponding clickhouse sink does not support [chproxy](https://github.com/ContentSquare/chproxy) so that can not ingest data to clickhouse.

But we have confirmed our chproxy setting is fine, another way can write data to clickhouse with it. But if we specify chproxy endpoint on vector setting, which can not write data to clickhouse and shows 400 bad requests.

So we want to check whether do not support chproxy on vector? thank you.

### Version

0.35.0

**Comment by jszwedko** [comment]
Hi @hueiyuan ,

Can you provide more details about why chproxy is returning 400s? I imagine chproxy might have some logs that could help. If Vector is able to successfully send to Clickhouse, but not chproxy, this could indicate a bug in chproxy rather than Vector.

**Comment by hueiyuan** [comment]
Hi, @jszwedko 
We have enabled debug mode but do not get more information. 
And based on vector request to chproxy headers, we found because our datetime is ISO-8601 Format. We also enable [date_time_best_effort](https://vector.dev/docs/reference/configuration/sinks/clickhouse/#date_time_best_effort), and this setting is by session.
So if we want to request to clickhouse with chproxy, the headers does not keep-alive so that can not parse related date time format to clickhouse with chproxy. 

Now workaround is we convert to unix_timestamp format in vector pipeline and ingest to clickhouse with chproxy, that's work.
